morphir/src/org/finos/morphir/runtime/TypedMorphirRuntimeDefs.scala

```diff
@@ -1,11 +0,0 @@
-package org.finos.morphir.runtime
-
-import org.finos.morphir.ir.Type.UType
-import org.finos.morphir.ir.Value.Value
-
-object TypedMorphirRuntimeDefs {
-  final type TypeAttribs  = scala.Unit
-  final type ValueAttribs = UType
-  type RuntimeValue       = Value[TypeAttribs, ValueAttribs]
-  type RuntimeDefinition  = Value.Definition[TypeAttribs, ValueAttribs]
-}
```

Comment:
@DamianReeves 

There's definitely been some miscommunication here. This change - the deletion of this file - was what I expected to see. Outside of that, my understanding was that everything that referenced types this file should be refactored to point to type definitions from `morphir/src/org/finos/morphir/ir/Value.scala` or `morphir/src/org/finos/morphir/ir/Type.scala` instead. Is that in line with your expectations?

One issue is anything that referenced `TypedMorphirRuntimeDefs.TypeAttribs` - that I did expect to be replaced with `scala.Unit`, given that there is no equivalent alias. However, complicating matters is that it looks like some places defined things as `Value[TypedMorphirRuntimeDefs.TypeAttribs, TypedMorphirRuntimeDefs.ValueAttribs]` , which was probably a pre-existing issue - those probably should have defined things as `TypedMorphirRuntimeDefs.RuntimeValue`, which could be replaced by `Value.TypedValue`. I'm not 100% sure, but I think that would eliminate all hard-coded uses of `scala.Unit` as a type in the runtime.